### PR TITLE
Add defaults to variable documentation

### PR DESCRIPTION
**Pull request by Alkali-Metal**
## What does this change?
Adding defaults to the user documentation for `variables`

## Requirements

 - [x] Only PG-rated language used (code, commits, etc.)
 - [x] Descriptive commit messages
 - [x] Changes have been tested
 - [x] Changes do not break any existing functionalities

### CODE REVIEW

File: docs/user/variables.md [hunk 1]
```diff
@@ -68,6 +68,17 @@ Change the output of variables. To use a modifier, append `|` and the modifier t
 
 Multiple modifiers may be chained, and will be evaluated from left to right.
 
+## `=` (Default)
+Any variable can be assigned a default by adding `=` and then whatever you want to be the default after it
```
Comment (2Cubed) [inline docs/user/variables.md]: Period after the sentence, and newline between text and \`\`\`, please? :)

**Review by 2Cubed — CHANGES_REQUESTED** [review]

File: docs/user/variables.md (hunk 1 above)
Comment (Alkali-Metal) [reply docs/user/variables.md]: Okey doke!

Comment (2Cubed) [inline docs/user/variables.md]: http://stackoverflow.com/q/43106938/7504176

Comment (2Cubed) [reply docs/user/variables.md]: Oh, and newline between `##` and `Any`, please. Oops.

**Review by 2Cubed — APPROVED** [review]

**Pull request merged**